src/io/flutter/server/vmService/DartVmServiceDebugProcess.java

```diff
@@ -35,78 +39,84 @@
 import org.dartlang.vm.service.VmService;
 import org.dartlang.vm.service.consumer.GetObjectConsumer;
 import org.dartlang.vm.service.consumer.VMConsumer;
-import org.dartlang.vm.service.element.*;
+import org.dartlang.vm.service.element.ElementList;
 import org.dartlang.vm.service.element.Event;
+import org.dartlang.vm.service.element.EventKind;
+import org.dartlang.vm.service.element.ExceptionPauseMode;
+import org.dartlang.vm.service.element.Isolate;
+import org.dartlang.vm.service.element.IsolateRef;
+import org.dartlang.vm.service.element.LibraryRef;
+import org.dartlang.vm.service.element.Obj;
+import org.dartlang.vm.service.element.RPCError;
+import org.dartlang.vm.service.element.Script;
+import org.dartlang.vm.service.element.ScriptRef;
+import org.dartlang.vm.service.element.Sentinel;
+import org.dartlang.vm.service.element.StepOption;
+import org.dartlang.vm.service.element.VM;
 import org.dartlang.vm.service.logging.Logging;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-import javax.swing.*;
-import java.awt.*;
-import java.awt.event.WindowEvent;
-import java.awt.event.WindowListener;
-import java.io.IOException;
-import java.nio.charset.StandardCharsets;
-import java.util.*;
-import java.util.List;
-
 public class DartVmServiceDebugProcess extends XDebugProcess {
   private static final Logger LOG = Logger.getInstance(DartVmServiceDebugProcess.class.getName());
 
   @Nullable private final ExecutionResult myExecutionResult;
   @NotNull private final DartUrlResolver myDartUrlResolver;
   @NotNull private final String myDebuggingHost;
   private final int myObservatoryPort;
-
-  private boolean myVmConnected = false;
-
   @NotNull private final XBreakpointHandler[] myBreakpointHandlers;
   private final IsolatesInfo myIsolatesInfo;
-  private VmServiceWrapper myVmServiceWrapper;
-
   @NotNull private final Set<String> mySuspendedIsolateIds = Collections.synchronizedSet(new THashSet<String>());
-  private String myLatestCurrentIsolateId;
-
   private final Map<String, LightVirtualFile> myScriptIdToContentMap = new THashMap<>();
   private final Map<String, TIntObjectHashMap<Pair<Integer, Integer>>> myScriptIdToLinesAndColumnsMap =
     new THashMap<>();
-
   private final int myTimeout;
   @Nullable private final VirtualFile myCurrentWorkingDirectory;
-  @Nullable protected String myRemoteProjectRootUri;
-
-
-  private VmOpenSourceLocationListener myVmOpenSourceLocationListener;
-
   @NotNull private final ObservatoryConnector myConnector;
-
   @NotNull
   private final ExecutionEnvironment executionEnvironment;
-
   @NotNull
   private final PositionMapper mapper;
-
+  @Nullable protected String myRemoteProjectRootUri;
+  private boolean myVmConnected = false;
   @NotNull private final OpenDartObservatoryUrlAction myOpenObservatoryAction =
     new OpenDartObservatoryUrlAction(null, () -> myVmConnected && !getSession().isStopped());
+  private VmServiceWrapper myVmServiceWrapper;
```

Comment:
can you layer the formatting only changes as a separate patch so it is easier to tell what is new what what has moved? I'm having trouble quickly telling what is the diff to review.

Reply:
I initially wrote "The main change is the addition of lines 555-558." I double-checked; that is the only change I intended to make. If you want, I can make a PR that just does the formatting. Once that's merged the diff viewer here will only show lines 555-558.